azkaban-common/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
 
   compile('com.google.guava:guava:13.0.1')
   compile('commons-collections:commons-collections:3.2.2')
-  compile('commons-dbcp:commons-dbcp:1.4')
+  compile('org.apache.commons:commons-dbcp2:2.1.1')
```

Comment:
Yes, maybe after next release so that we can isolate potential problems with updates.